ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/CustomerCenterOptions.kt

```diff
@@ -0,0 +1,42 @@
+package com.revenuecat.purchases.ui.revenuecatui.customercenter
+
+import com.revenuecat.purchases.customercenter.CustomerCenterListener
+
+class CustomerCenterOptions internal constructor(
+    val listener: CustomerCenterListener?,
+) {
+
+    constructor(builder: Builder) : this(
```

Comment:
Hmm should the constructor here be `internal`? We should only allow building an instance of the options through the builder `build` method right?